providers/fips/fipsprov.c

```diff
@@ -37,17 +37,19 @@ static OSSL_FUNC_provider_get_params_fn fips_get_params;
 static OSSL_FUNC_provider_query_operation_fn fips_query;
 
 /* Locale object accessor functions */
-#ifdef OPENSSL_SYS_MACOSX
```

Comment:
We'd normally do this:
```c
#if defined(OPENSSL_SYS_MACOSX)
```
Although in this case leaving it as `#ifdef` would be acceptable too.

Reply:
What?  Since when have we preferred `#if defined(blah)` over `#ifdef blah`?  We use the `#ifdef` form for single macro checks all over the place!

Reply:
I'm not getting I the middle of this. Reverting.